specification/resources/resource-manager/Microsoft.Resources/stable/2024-04-01/resources.json

```diff
@@ -6456,6 +6514,24 @@
             "changeType"
           ],
           "description": "List of resource changes predicted by What-If operation."
+        },
+        "potentialChanges": {
```

Comment:
What's the reason to add `potentialChanges` here?

Reply:
That's to account for Jonny's update, for example when a resource might get updated but we are not fully certain,